bin/dump.sh

```diff
@@ -157,6 +160,28 @@ combine_pkcs12() {
   fi
 }
 
+convert_keys_to_rsa() {
+  if [[ "${CONVERT_KEYS_TO_RSA}" != yes ]]; then
+    return
+  fi
+
+  if [[ -z "${DOMAIN}" || "${#DOMAINS[@]}" -gt 1 ]]; then
+    local outputdir_subdirs=(${outputdir}/*/)
+    for subdir in "${outputdir_subdirs[@]}"; do
+      local i=$(basename "${subdir}" /)
+      if [[ -f ${outputdir}/${i}/cert.pem && -f ${outputdir}/${i}/key.pem ]]; then
+        log "Converting key for domain ${i} to RSA key file"
+        openssl rsa -in ${outputdir}/"${i}"/key.pem -out ${outputdir}/"${i}"/rsakey.pem
+      fi
+    done
+  else
+    if [[ -f ${outputdir}/cert.pem && -f ${outputdir}/key.pem ]]; then
```

Comment:
```suggestion
    if [[ -f "${outputdir}/${certificate_file}" && -f "${outputdir}/${privatekey_file}" ]]; then
```

See previous review comment regarding #78.